commands/bridge_push.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/MichaelMure/git-bug/bridge"
+	"github.com/MichaelMure/git-bug/bridge/core"
+	"github.com/MichaelMure/git-bug/cache"
+	"github.com/MichaelMure/git-bug/util/interrupt"
+)
+
+func runBridgePush(cmd *cobra.Command, args []string) error {
+	backend, err := cache.NewRepoCache(repo)
+	if err != nil {
+		return err
+	}
+	defer backend.Close()
+	interrupt.RegisterCleaner(backend.Close)
+
+	var b *core.Bridge
+
+	if len(args) == 0 {
+		b, err = bridge.DefaultBridge(backend)
+	} else {
+		b, err = bridge.LoadBridge(backend, args[0])
+	}
+
+	if err != nil {
+		return err
+	}
+
+	// TODO: by default export only new events
+	err = b.ExportAll(time.Time{})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+var bridgePushCmd = &cobra.Command{
+	Use:     "push [<name>]",
+	Short:   "Push updates.",
```

Comment:
It would be neat to have a better doc, what is actually exported, what is not and why ....

Reply:
Shall we wait until `--allow-origins` and `--since` flags are implemented ? i think it's necessary to document this command since they have big impact on how export is done

Reply:
As long as it's there eventually :)

Reply:
Will do in #167 ! :)